src/dd-draggable.ts

```diff
@@ -317,7 +317,9 @@ export class DDDraggable extends DDBaseImplement implements HTMLElementExtendOpt
     const style = this.helper.style;
     const { scaleX, scaleY } = Utils.getScaleForElement(this.helper);
     const transformParent = Utils.getContainerForPositionFixedElement(this.helper);
-    const transformParentRect = transformParent.getBoundingClientRect();
+    // We need to be careful here as the html element actually also includes scroll
+    // so in this case we always need to ignore it
+    const transformParentRect = transformParent === document.documentElement ? { top: 0, left: 0 } : transformParent.getBoundingClientRect();
```

Comment:
isn't this assuming here that the entire doc is scrollable, but it could be any parent div in between. I thought we had code to detect scrolled parent and take that into content...

Reply:
yes actually you are right the issue is bigger than that. I will fix it 😄